src/os_posix/os_union_fs.c

```diff
@@ -11,13 +11,14 @@
 
 #define WT_UNION_FS_TOMBSTONE_SUFFIX ".deleted"
 
-static int __union_merge_with_next_extents(WT_SESSION_IMPL *session, WT_UNION_ALLOC_LIST *extent);
 static int __union_fs_file_read(
   WT_FILE_HANDLE *file_handle, WT_SESSION *wt_session, wt_off_t offset, size_t len, void *buf);
 static int __union_fs_file_size(
   WT_FILE_HANDLE *file_handle, WT_SESSION *wt_session, wt_off_t *sizep);
 
-#define EXTENT_END(ext) ((ext)->off + (wt_off_t)(ext)->len)
+/* FIXME - Make these static functions? */
```

Comment:
Add comment explaining -1.

Reply:
Added in [244bf44](https://github.com/wiredtiger/wiredtiger/pull/11268/commits/244bf444361185894d3c45215bcb0b91e9357af1)